.github/workflows/knative-boilerplate.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 The Knative Authors.
```

Comment:
Should this boilerplate be added in a different PR? Seems odd for this PR to include it

Reply:
(Actually shouldn't this stuff be added by the automation? 🤔)

Reply:
I literally just copied everything from the `kn-plugin-sample` repo (as per [these](https://github.com/knative-sandbox/kn-plugin-sample/blob/main/BLUEPRINT.md#init) instructions)

Reply:
Everything in commit `dcd8dc1` should be exactly the same as what was in the sample repo. My updates are in the following two commits.
(just trying to explain _what_ was done, not justifying that it was the right thing :) )

Reply:
yeah I haven't written a cli plugin before so I'm not sure, but I think it probably makes sense to have the .github folder be synced via knobots, similar to how e.g. kn-plugin-diag works (see https://github.com/knative-sandbox/knobots/blob/main/repos.yaml#L201). Maybe @maximilien or @rhuss can confirm?

Either way, I think it'd make sense to land an initial PR first with just the boilerplate, and then this one will be a bit easier to review since it'll just be the new stuff.